packages/server/src/api/storage/sql/sql.js

```diff
@@ -92,6 +92,31 @@ function createSequelize(options) {
     });
   }
 
+  if (options.sqlSocketPath) {
+    if (
+      !options.sequelizeOptions ||
+      !options.sequelizeOptions.database ||
+      !options.sequelizeOptions.username ||
+      !options.sequelizeOptions.password
+    )
+      throw new Error(
+        `Cannot use ${dialect} with socketPath without database, username, or password`
+      );
```

Comment:
```suggestion
    ) {
      throw new Error(
        `Cannot use ${dialect} with socketPath without database, username, or password`
      );
    }
```

Reply:
What is the suggestion here? (I can't see the difference between what I added and what you suggested @patrickhulce )

Reply:
it's just adding braces around the throw :)